Transbank/Common/Options.cs

```diff
@@ -9,6 +9,8 @@ public class Options
         private string _apiKey;
         private IIntegrationType _integrationType;
         private RequestService _requestService;
+        private const int DefaultTotalTimeout = 60 * 10;
+        private int _totalTimeout;
```

Comment:
perhaps totalTimeout is not so descriptive. That is, is there any timeout that is not total?
This point is just a suggestion.